style.css

```diff
@@ -1,4 +1,234 @@
-:root {
-	--white: #FFFCFC;
+/* Font families available from Typekit ID edn5kex
+font-family: fira-sans, sans-serif;
+font-family: base-900-sans, sans-serif;
+font-family: rift, sans-serif;
+font-family: fira-sans, sans-serif; font-weight: 400; font-style: normal;
+font-family: fira-sans, sans-serif; font-weight: 400; font-style: italic;
+font-family: fira-sans, sans-serif; font-weight: 700; font-style: normal;
+font-family: fira-sans, sans-serif; font-weight: 700; font-style: italic;
+font-family: base-900-sans, sans-serif; font-weight: 900; font-style: italic;
+font-family: rift, sans-serif; font-weight: 600; font-style: normal;
+*/
+
+/* Style variables */
+body {
+	background-color: #FFFCFC;
+	color: #171716;
+}
+
+h1 {
+	font-family: rift, sans-serif;
+	font-weight: 600;
+	font-style: normal;
+	font-size: 64px;
+}
+
+h2 {
+	font-family: rift, sans-serif;
+	font-weight: 600;
+	font-style: normal;
+	font-size: 48px !important;
+}
+
+h3 {
+	font-family: base-900-sans, sans-serif;
+	font-weight: 900;
+	font-style: italic;
+	font-size: 36px;
+}
+
+h4 {
+	font-family: base-900-sans, sans-serif;
+	font-weight: 900;
+	font-style: italic;
+	font-size: 24px;
+}
+
+p {
+	font-family: fira-sans, sans-serif;
+	font-weight: 400;
+	font-style: normal;
+	font-size: 18px;
+}
+
+.site-header {
+	padding-top: 1rem;
+	padding-bottom: 1rem;
+}
```

Comment:
@phillipburger @rickalday I'd like to see `justify-content: center;` here, and perhaps let's make it more specific by making it target `#masthead.site-header`. This way the menu will be vertically centred with the tagline in play as well.

Reply:
Let me check this now.

Reply:
For the vertical centering, I find that `align-items: center;` seems to do the trick, I have added to the local CSS for validation now and will check it into this PR